cinder/files/liberty/cinder.conf.controller.Debian

```diff
@@ -87,6 +87,8 @@ control_exchange = cinder
 notification_driver = messagingv2
 {%- endif %}
 
+# Available queries for non-admin user
+query_volume_filters_opt = name,status,metadata,availability_zone,bootable
```

Comment:
default is query_volume_filters_opt = name,status,metadata,availability_zone in cinder source code